ci/gpu/build.sh

```diff
@@ -66,6 +66,7 @@ conda install -c nvidia -c rapidsai -c rapidsai-nightly -c conda-forge -c defaul
       distributed>=2.12.0 \
       dask-cudf=${MINOR_VERSION} \
       dask-cuda=${MINOR_VERSION} \
+      scikit-learn>=0.21 \
```

Comment:
Why is scikit-learn being added as a dependency?

Reply:
Don't we need it here since I'm relying on scikit learn for my tests ?

Reply:
Is the dependency only for tests?

Reply:
Yes